RELEASE-NOTES.txt

```diff
@@ -1,4 +1,14 @@
-*** PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
+// ## Please follow the below format. See pdcxQM-Km-p2 for more details.
+// 
+// ```
+// * [<priority indicator, more stars = higher priority>] <optional app area>: <description> [<PR URL>]
+// ```
+// 
+// - The priority indicator usually ranges from `*` (small impact/importance) to `***` (high importance), and will help pick the most important items when wordsmithing the final, editorialized release notes from the items listed here.
+// - The app area prefix before the description is optional, but can help categorize features. Examples include: `Block Editor:`, `Reader:`, etc.
+// - If your change is not user-facing (and thus should not be included in the final public-facing release notes) but still needs to be included in the Call for Testing,
+//   so that it is still beta-tested during that sprint (e.g. for features behind feature flags), use `[internal]` in place or after the priority indicator.
+//
```

Comment:
Note: Our tooling (especially the action that will add a new entry after those comments but before the first section) currently accepts either `***` or `//` to mark the top lines of the file as a comment.

Reply:
I've made this a bit verbose to provide more info… but feel free to also consider going the other way, by making it more succinct but directly point to the P2 as source of truth to provide all the details… up to you.